sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/CosmosDiagnosticsSystemUsageSnapshot.java

```diff
@@ -28,6 +35,12 @@ public final class CosmosDiagnosticsSystemUsageSnapshot {
         this.usedMemory = usedMemory;
         this.availableMemory = availableMemory;
         this.availableProcessors = availableProcessors;
+        ImmutableMap.Builder<String, String> builder = ImmutableMap.<String, String>builder();
+        builder.put("CPU", systemCpuLoad);
+        builder.put("Memory used", usedMemory);
+        builder.put("Memory available", availableMemory);
+        builder.put("Processor count", String.valueOf(availableProcessors));
```

Comment:
nit: 
use a static/cached  `String.valueOf(availableProcessors)`, or, if you decide to use Object, cache `Integer`

Reply:
Changed to Map<String, Object>